File: app/src/main/java/de/tum/in/tumcampusapp/models/Faculty.java
package de.tum.in.tumcampusapp.models;

/**
 * Presents the faculty model that is used in fetching the facultyData from server
 */
public class Faculty {
    private String faculty;
    private String name;

    public Faculty(String id, String name) {
        this.faculty = id;
        this.name = name;
    }

    public String getId() {
        return faculty;
    }

    public void setId(String id) {
        this.faculty = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/models/tumcabe/Faculty.kt
package de.tum.`in`.tumcampusapp.models.tumcabe

import android.arch.persistence.room.Entity

/**
 * Presents the faculty model that is used in fetching the facultyData from server
 * @param faculty the id of the faculty
 */
@Entity
data class Faculty(var faculty: String = "",
                   var name: String = "")